components/odh-notebook-controller/controllers/suite_test.go

```diff
@@ -129,6 +130,16 @@ var _ = BeforeSuite(func() {
 	})
 	Expect(err).NotTo(HaveOccurred())
 
+	if kubeconfigPath, found := os.LookupEnv("DEBUG_WRITE_KUBECONFIG"); found {
```

Comment:
> Maybe you could put some note in the readme or somewhere? Just state that this property exists, what it is for and how it can be utilized. So that anyone can see easily? 🙂

Yeah, good point. Does the env variable name make sense? And should there be a log message about creating/not creating the kubeconfig file?

Reply:
No problem with the variable name from my side. Also, log message is a good idea - would be nice to print it including the path where this file will be written out.

Reply:
Added both